### DISCUSSION TITLE
How to update a single property on a record

### DISCUSSION THREAD

**Discussion by revenz** [litedb-org/LiteDB#2291] [Q&A]
In normal SQL you would do 
```
update myTable set property = @0
```
or
```
update myTable set property = @value
```
and pass a SqlCommandParameter("value", "myvalue");

(or something along those lines, been a while since I used SqlCommand stuff directly).

How would I go about doing this in LiteDB?

I think its
```
db.Execute("update mytable set property = ????")
```
Just not sure how to use a parameter here.  Or is this completely wrong?

**Comment by LMeinhardt** [comment]
Hey revenz,

From one of the examples on the performance benchmark, he uses the following code:

   public override void Update(int count)
        {
            _db.BeginTrans();

            for (var id = 1; id <= count; id++)
            {
               // find the correspondent content (id) 
               var doc = _col.FindById(id);
               // Set the content field (in this case the field Description
               doc.Description = Helper.LoremIpsum(15, 15, 3, 3, 4);
               // Update the record
                _col.Update(doc);
            }
           // Finally commit the content -  very straight forward
            _db.Commit();
        }
